compiler/plugins/target/CUDA/CUDATarget.cpp

```diff
@@ -242,14 +243,26 @@ static std::string produceGpuImage(const CUDAOptions &options,
   return ptxImage;
 }
 
-static void dumpBitcodeToPath(StringRef path, StringRef baseName,
-                              StringRef suffix, StringRef extension,
-                              llvm::Module &module) {
-  llvm::SmallVector<char, 0> data;
-  llvm::raw_svector_ostream ostream(data);
-  llvm::WriteBitcodeToFile(module, ostream);
-  dumpDataToPath(path, baseName, suffix, extension,
-                 StringRef(data.data(), data.size()));
+static void dumpLLVMModuleToPath(StringRef path, StringRef baseName,
+                                 StringRef suffix, StringRef extPrefix,
+                                 llvm::Module &module) {
+  // Dump disassembly to path.
+  llvm::SmallVector<char, 0> textData;
+  llvm::raw_svector_ostream textOstream(textData);
+
+  module.print(textOstream, nullptr);
+  llvm::Twine textExtension = llvm::Twine(extPrefix) + ".ll";
```

Comment:
We can't use twines for local variables: https://llvm.org/docs/ProgrammersManual.html#llvm-adt-twine-h

Either pass it as a function parameter or materialize as a `std::string`